### ISSUE TITLE
undefined method `members' for nil:NilClass

### ISSUE THREAD

**Issue by monkomonk** [brigade/scss-lint#168]
Apologies in advanced if this is an issue with my files, I'm relatively new to ruby gems. 

Standard command

```
 scss-lint scss/ 
```

returns this 

```
SCSSLint::Linter::MergeableSelector raised unexpected error linting file scss/_page-wishlist.scss: 'undefined method `members' for nil:NilClass'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/linter/mergeable_selector.rb:51:in `single_rule?'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/linter/mergeable_selector.rb:39:in `nested?'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/linter/mergeable_selector.rb:30:in `block in find_mergeable_node'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/linter/mergeable_selector.rb:28:in `each'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/linter/mergeable_selector.rb:28:in `find'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/linter/mergeable_selector.rb:28:in `find_mergeable_node'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/linter/mergeable_selector.rb:10:in `block in check_node'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/linter/mergeable_selector.rb:7:in `each'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/linter/mergeable_selector.rb:7:in `each_with_object'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/linter/mergeable_selector.rb:7:in `check_node'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/sass-3.3.7/lib/sass/tree/visitors/base.rb:36:in `visit'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/linter.rb:115:in `visit'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/linter.rb:125:in `block in visit_children'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/linter.rb:123:in `each'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/linter.rb:123:in `visit_children'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/sass-3.3.7/lib/sass/tree/visitors/base.rb:36:in `block in visit'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/linter/mergeable_selector.rb:19:in `check_node'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/sass-3.3.7/lib/sass/tree/visitors/base.rb:36:in `visit'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/linter.rb:115:in `visit'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/linter.rb:18:in `run'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/runner.rb:43:in `block in find_lints'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/runner.rb:38:in `each'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/runner.rb:38:in `find_lints'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/runner.rb:22:in `block in run'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/runner.rb:21:in `each'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/runner.rb:21:in `run'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/lib/scss_lint/cli.rb:98:in `run'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/bin/scss-lint:7:in `block in <top (required)>'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/bin/scss-lint:5:in `tap'
/Users/Ollie/.rvm/gems/ruby-2.1.1@global/gems/scss-lint-0.24.0/bin/scss-lint:5:in `<top (required)>'
/Users/Ollie/.rvm/rubies/ruby-2.1.1/bin/scss-lint:23:in `load'
/Users/Ollie/.rvm/rubies/ruby-2.1.1/bin/scss-lint:23:in `<main>'
/Users/Ollie/.rvm/gems/ruby-2.1.1/bin/ruby_executable_hooks:15:in `eval'
/Users/Ollie/.rvm/gems/ruby-2.1.1/bin/ruby_executable_hooks:15:in `<main>'
Report this bug at https://github.com/causes/scss-lint/issues

```

see last line as to why I'm posting here
